concent_api/core/views.py

```diff
@@ -661,6 +665,16 @@ def validate_id_value(value, field_name):
         raise Http400("{} cannot be longer than {} chars.".format(field_name, MESSAGE_TASK_ID_MAX_LENGTH))
 
 
+def validate_public_key(value, field_name):
+    assert isinstance(field_name, str)
+
+    if not isinstance(value, bytes):
+        raise Http400("{} must be string.".format(field_name))
+
+    if len(value) != GOLEM_PUBLIC_KEY_LENGTH:
+        raise Http400("{} length cannot be different than {} chars.".format(field_name, GOLEM_PUBLIC_KEY_LENGTH))
```

Comment:
Maybe `"The length of {} must be exactly {} characters."`?

Reply:
:+1: